smartcontractkit/external-adapters-js
# Linear Finance EA fixes

## Closes 31321

## Description

This PR addresses the following issues when running the Linear Finance EA against some source EAs

 - `CRO-USD` pair not supported when requesting from the Coingecko EA
 - `SAND-USD` and `IMX-USD` pairs not supported when requesting from Nomics EA
 -  Failed requests to Coinpaprika when sending many pairs at once

## Changes

- Add token overrides to Nomics and Coingecko
- Batch requests when requesting pairs from Coinpaprika



## Steps to Test

1. Run Linear Finance EAs and the Coingecko, Nomics and Coinpaprika EAs
2. Send requests to the Linear Finance EA and source the prices from the above EAs.
3. Verify successful responses
4. 
## Quality Assurance
